src/Prestige.ts

```diff
@@ -52,6 +53,9 @@ export function prestigeAugmentation(): void {
   // We must kill all scripts before doing anything else.
   prestigeWorkerScripts();
 
+  // Close coding contract modal
+  CodingContractEventEmitter.emit({ type: "close" });
```

Comment:
This function is just a grab-bag of stuff, but still it has a *slight* but of structure to it. I've suggest moving this to right after LogBoxEvents.Clear() (which is with other terminal stuff) as the most natural place.